tests/integration/maybe-in-element-test.js

```diff
@@ -49,4 +49,20 @@ module('Integration | maybe-in-element', function(hooks) {
     assert.dom('#test-destination-element').containsText('Some text', 'The content has been rendered in the destination element');
     assert.dom('#original-placement').doesNotContainText('Some text', 'The content wasn\'t rendered in the original location');
   });
+
+  test('It preserves context for actions', async function(assert) {
+    assert.expect(0);
+    this.renderInPlace = false;
+    this.actions = {
+      test() {}
+    };
```

Comment:
does it work with `this.set('test', () => {})` instead?

Reply:
Not sure, but if I move the action helper outside of the `maybe-in-element` block, then it works fine! So I think something (no idea what) with the transform is not working quite right. 

I saw that exception in my own code base, so I was happy to actually be able reproduce it in a local test, rather than somehow make it pass! 😉